app/controllers/users_controller.rb

```diff
@@ -1,4 +1,6 @@
 class UsersController < ApplicationController
```

Comment:
Unrelated to this change, but worth noting that moving this into the account reset namespace may be a good refactor. Confusing to have this controller out here, makes you think this is the controller for the "delete account" button on the account page.

Reply:
Ah, I'm misreading this. This is actually for cancelling during account setup, ignore the above